### PR TITLE
Index page (index.php) for #2972

### PR DESCRIPTION
**Pull request by Matslom**
FOR #2972

**templates;**
*index*
- [x]  index
- [x]  index_birthdays
- [x]  index_birthdays_birthday
- [x]  index_boardstats
- [x]  index_logoutlink
- [x]  index_stats
- [x]  index_statspage
- [x]  index_whosonline
- [x]  index_whosonline_memberbit

```diff
+ index -> index/index.twig
- index_birthdays
- index_birthdays_birthday
+ index_boardstats -> index/boardstats.twig
- index_logoutlink
- index_stats
- index_statspage
- index_whosonline
- index_whosonline_memberbit
```

### CODE REVIEW

File: inc/views/base/index/boardstats.twig [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                    {{ lang.stats_newestuser|raw }}<br />
+                    {{ lang.stats_mostonline }}
+                </span>
+                </td>
+            </tr>
+        {% endif %}
+    <tr>
+        <td class="tfoot" style="text-align: right">
+		<span class="smalltext">
+            {% if mybb.user.uid != 0 %}
```
Comment (euantorano) [inline inc/views/base/index/boardstats.twig]: Indentation here looks a little bit weird.

**Comment by Matslom** [comment]
@euantorano I had to remove forumbit templates from my todo list in this PR. They are too complex and time consuming and I can't handle them now.

**Comment by euantorano** [comment]
@Matslom No problem. I'm going to leave this open for now then until the forum bit templates are done - otherwise it might make it hard to test other PRs without an index page to navigate into forums and threads 😉

**Comment by Matslom** [comment]
It may stay open, but I've add forums raw html, so they are rendered from old templates.

**Pull request closed without merging**